src/demo/colors/Effects.tsx

```diff
@@ -13,6 +13,11 @@ const misc = [
         title: 'Shadow',
         description: 'Shadow for everything that might have it.',
     },
+    {
+        name: 'sfx-shadow-light',
+        title: 'Shadow Light',
+        description: 'Shadow for card size m.',
```

Comment:
It's better not to mention specific components here, let's say something like that: *Lighter version of shadow*